contracts/src/v0.8/automation/dev/v2_3/AutomationRegistryBase2_3.sol

```diff
@@ -324,11 +320,13 @@ abstract contract AutomationRegistryBase2_3 is ConfirmedOwner {
     uint32 performGas;
     uint32 maxValidBlocknumber;
     IAutomationForwarder forwarder;
-    // 0 bytes left in 1st EVM word - not written to in transmit
+    // 0 bytes left in 1st EVM word - read in transmit path
     uint96 amountSpent;
     uint96 balance;
     uint32 lastPerformedBlockNumber;
     // 2 bytes left in 2nd EVM word - written in transmit path
+    IERC20 billingToken;
```

Comment:
curious, how do you know the size of this object, e.g. IERC20 and BillingConfig struct